example/react-router-cloudflare-example/package.json

```diff
@@ -5,7 +5,7 @@
   "sideEffects": false,
   "type": "module",
   "scripts": {
-    "build": "NODE_ENV=production react-router build",
+    "build": "NODE_ENV=production pnpm fbtee:manifest && pnpm fbtee:collect && pnpm fbtee:translate && react-router build",
```

Comment:
Shouldn't `pnpm fbtee:translate && react-router build` suffice here, as it anyways works on a different set of files then the ones created by manifest and collect?

Reply:
The translate command needs `.source_strings.json`.